disable the build and installation of the documentation of mayavi
it can cause issues in environments with deprecated numpy API v1.7
author: Alex Domingo (Vrije Universiteit Brussel)
--- setup.py.orig	2018-06-05 13:54:38.000000000 +0200
+++ setup.py	2020-12-11 17:18:08.541646210 +0100
@@ -266,12 +266,6 @@
     def run(self):
         build_tvtk_classes_zip()
         build.build.run(self)
-        self.run_command('gen_docs')
-        try:
-            self.run_command('build_docs')
-        except:
-            log.warn("Couldn't build documentation:\n%s" %
-                     traceback.format_exception(*sys.exc_info()))
 
 
 class MyDevelop(develop.develop):
@@ -435,8 +429,6 @@
             'clean': MyClean,
             'develop': MyDevelop,
             'install_data': MyInstallData,
-            'gen_docs': GenDocs,
-            'build_docs': BuildDocs,
             },
         description='3D scientific data visualization library and application',
         download_url=('https://www.github.com/enthought/mayavi'),
